snewpdag/values/Hist1D.py

```diff
@@ -57,6 +78,12 @@ def fill(self, x, weight=1.0):
     self.sum2 += np.sum(v*v)
     self.count += weight * v.size
 
+  def add(self, x, weight=1.0):
```

Comment:
actually, add() is a synonym of fill().  Unfortunately here changing the name to something like add_times() doesn't help, because Hist1D doesn't actually care what its horizontal axis represents.  So may need to keep it as add() for Hist1D as well as TimeSeries, since they need to have the same interface.